docs/blog/posts/move-to-litellm-proxy.md

```diff
@@ -0,0 +1,25 @@
+---
+authors:
+  - Amir
+category:
+  - Architecture
+
+date: 2024-04-05
+---
+# Model calls made flexible
+
+Instead of being handled directly in our code, our model calls are now routed through an LiteLLM Proxy server, which lets use change models on the fly and have retries, fallbacks, budget tracking, and more.
+
+<!-- more -->
+
+## Why change?
+
+[LiteLLM Proxy](https://litellm.vercel.app/docs/simple_proxy) introduces a streamlined approach for handling various large language models (LLMs) through a single interface. We can now manage model endpoints and configurations via a proxy server, replacing the previous method of hard-coding them into the environment of our app.
+
+This setup not only simplifies the codebase by centralizing the model configurations but also provides the flexibility to switch or update models without altering the core application logic. We can even add and remove models through the proxy's API.
```

Comment:
Can you say something about multitenancy? Multiple instances of AAQ can call the proxy server